scripts/create_pdf.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+"""
+python script to create PDFs for Startwords articles using the DocRaptor API.
+
+To use:
+
+- install python dependencies: `pip install docraptor bs4 requests`
+- create a DocRaptor account and get an API key https://docraptor.com
+- set your API key as environment variable DOCRAPTOR_API_KEY
+- run the script with a Startwords article url and an output filename, e.g.
+  `create_pdf.py https://startwords.cdh.princeton.edu/issues/3/mapping-latent-spaces/ -o
+  startwords-3-mapping-latent-spaces.pdf`
+
+The DocRaptor API allows unlimited test PDFs, which are watermarked; creating
+a test PDF is the default behavior for this script. When you are ready to
+create a final PDF, use the `--no-test` flag to turn test mode off.
+
+To generate PDFs for all the feature articles in one Startwords issue,
+use the `--issue` flag and run with the issue url, e.g.,
+`create_pdf.py https://startwords.cdh.princeton.edu/issues/3/mapping-latent-spaces/ --issue`
+
+PDFs will be named according to current Startwords PDF naming conventions.
+
+"""
```

Comment:
Nice, very clear documentation, and it's cool it accepts (already legible) URLs as input